kubetools/kubernetes/api.py

```diff
@@ -41,11 +42,40 @@ def _get_k8s_apps_api(env):
     return client.AppsV1Api(api_client=api_client)
 
 
-def _get_k8s_batch_api(env):
+def check_if_cronjob_compatible(env):
+    settings = get_settings()
+
+    api_core = _get_k8s_core_api(env)
+    list_node = api_core.list_node().items
+    k8s_version = "1.21.0"
+    lowest_version = "1.21.0"
+
+    # Get the lowest kubelet version of all the nodes of the Cluster
+    for node in list_node:
+        kubelet_version = node.status.node_info.kubelet_version.replace('v', '')
+        if version.parse(kubelet_version) < version.parse(lowest_version):
+            lowest_version = kubelet_version
+
+    if version.parse(lowest_version) >= version.parse(k8s_version):
+        settings.IS_CRONJOB_COMPATIBLE = True
+
+    return settings.IS_CRONJOB_COMPATIBLE
```

Comment:
```suggestion
    return version.parse(lowest_version) >= version.parse(k8s_version):
```

Also, rename `k8s_version` into `required_k8s_version`

Reply:
Both are fixed in this commit https://github.com/EDITD/kubetools/pull/131/commits/151a47065791870b477884869c2306d1589db324